imagetest/fixtures.go

```diff
@@ -240,6 +241,19 @@ func (t *TestVM) EnableSecureBoot() {
 	}
 }
 
+// ChangeNicTypeToGVNIC changes the type of vNIC to be used to GVNIC
+func (t *TestVM) ChangeNicTypeToGVNIC() {
```

Comment:
it's a nitpick, but i feel 'change' implies amending something that exists, but we are actually setting this at launch time. maybe 'UseGVNIC' or similar?

Reply:
Agreed, UseGVNIC sounds more appropriate here